business/views.py

```diff
@@ -11,12 +12,25 @@ def sponsor(request):
 
 
 def jobs(request):
-    offers_objs = Offer.objects.filter(
-        Q(ends_at__isnull=True) | Q(ends_at__gt=timezone.now()),
-        is_visible=True,
-        starts_at__lte=timezone.now(),
-    ).order_by("-created_at")
-    return render(request, "jobs.html", {"offers": offers_objs})
+    type = request.GET.get("type", "")
```

Comment:
No need to set default, so better to `type = request.GET.get("type")` and then `type is not None`.